war/src/main/less/base/style.less

```diff
@@ -1422,6 +1462,21 @@ DIV.to-be-removed { display: none; }
 
 /* ========================= Other form related CSS ========================= */
 
+.form-container {
+  max-width: 100%;
+}
+
+.optionalBlock-container >.form-container,
+.radioBlock-container >.form-container {
+  margin-left: 1rem;
+  border-left: 1px solid @color-primary;
```

Comment:
is this border intended to stay? @fqueiruga 

great to see the nesting fixed

![image](https://user-images.githubusercontent.com/21194782/78453882-7cd42b80-768c-11ea-8378-30fe486e84fa.png)

Reply:
I'd argue that we can use a visual indication of the nesting. I'd keep the border at least on the WIP PR. It's always nice to add touches of color to the UI

FWIW this is blue is intented to be the primary color. There will be buttons with it, hyperlinks will have it on hover, etc. We can always swap it later